inspire/base/searchext/mappings/jobshidden.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
In particular, Jobs are moving from the Jobs collection to a hidden one, simply based on a check over the metadata. So we might want to preserve this easyness.